.github/workflows/linter.yml

```
@@ -0,0 +1,18 @@
+---
+on:
+  - pull_request
+  - push
+
+name: linter
+
+jobs:
+  linter:
+    uses: php-forge/actions/.github/workflows/super-linter.yml@v2
+    env:
```

actionlint (1.7.7)

when a reusable workflow is called with "uses", "env" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "linter"

(syntax-check)